src/components/Shared/SocialShareModal.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+// @ts-ignore
+import { Modal } from "react-router-modal";
+import { copyToClipboard } from "lib/util";
+import Tooltip from "rc-tooltip";
+import * as css from "./SocialShareModal.scss";
+
+interface IState {
+  showCopiedFeedback: boolean;
+}
+
+interface IProps {
+  closeHandler: (event: any) => void;
+  url: string;
+}
+
+export default class SocialShareModal extends React.Component<IProps, IState> {
+
+  constructor(props: IProps) {
+    super(props);
+
+    this.selectTwitter = this.selectTwitter.bind(this);
```

Comment:
FYI there is another way around having to do this binding, using arrow functions like this: private showCopiedFeedback = (event: any) => {...} or if you need to pass a variable to the handler showCopiedFeedback = (param: number) => (event: any) => {...}

https://medium.com/silesis/handle-events-in-react-with-arrow-functions-ede88184bbb